python/artm/regularizers.py

```diff
@@ -66,29 +67,63 @@ def _update_config(self, obj, first=False):
 
 
 class Regularizers(object):
-    def __init__(self, master):
+    def __init__(self, master, force_add=False):
```

Comment:
Ah, I see --- I haven't thought about operator[].
But maybe we can agree that `operator[]` always behaves as if `force_add=True`, while `add` always behaves as `force_add=False`? I think this makes sense (add typically means "add something new", so it should raise an error if it already exists; while using operator[] involves assignment operator -- which naturally overwrites previous value).